sdk/storage/CMakeLists.txt

```diff
@@ -18,11 +18,11 @@ set (AZURE_STORAGE_BLOB_HEADER
     inc/common/crypt.hpp
     inc/common/xml_wrapper.hpp
     inc/blobs/blob.hpp
+    inc/blobs/blob_service_client.hpp
+    inc/blobs/blob_container_client.hpp
     inc/blobs/blob_client.hpp
     inc/blobs/block_blob_client.hpp
-    inc/blobs/blob_container_client.hpp
     inc/blobs/blob_client_options.hpp
```

Comment:
`blob_client_options` should now be renamed to `blob_options`, given it contains client options and request options for the entire blob service.

Reply:
okay